### DISCUSSION TITLE
show version number in cli

### DISCUSSION THREAD

**Discussion by tvld** [feathersjs/feathers#3039] [Ideas]
Just an idea: when I run ``feathers generate service`` I like to be certain I use a certain version. Sometimes, with npm global and what not, we might use the wrong one, so maybe show it by default:

```bash
$ feathers generate service
Feathers 5.0.0-pre.34
? What is the name of your service? teams
...
```

**Comment by daffl** [comment]
I think part of the problem at the moment is that transitory period where the v4 CLI was installed globally vs in v5 apps where the CLI is a local development dependency by default. When going with the v5 process of `npm create feathers my-app` and then in `my-app` using `npx feathers` it will always use the proper local dependency first.

Essentially a global `feathers` command will no longer be available at all. We could do what some other tools are doing and alias the global to the local command and throw an error if a local version is not available.

**Reply by tvld** [reply]
The alias is a  perfect solution. I always feel a bit ashamed when I stumble upon some error like this. Usually as part of investigation of another error :-))))
